.gitignore

```diff
@@ -8,6 +8,8 @@
 _obj
 _test
 
+test/integration/licenses/
```

Comment:
I'm fixing the `licenses` dir not being removed with this PR here: https://github.com/kubernetes/minikube/pull/15376

So could you remove this line and instead just run `rm -rf test/integration/licenses` assuming your on Unix

Reply:
Okay will do!